renovate.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
```

Comment:
> With your current configuration, Renovate will create 47 Pull Requests

.... how about not lol
Maybe we should do a manual update first?

> Update dependency typescript to ~4.9.0

this sounds like a bad idea, I'd like to keep a single version of TypeScript used across the repo (while not technically a necessity, prevents a bunch of headaches debugging issues). In effect that means we are bounded by the minimum of fast vs angular's typescript version.

> Update dotnet monorepo to v7 (major)

Not sure that's a great idea, we effectively just have one blazor client with BTS that I am aware of, are thy on .net 7 yet?

Maybe it's just the sheer number of updates here but my first reaction is I'd rather it handle 3rd party dependencies by bumping within the semver range of the package json and not try and update to the latest major versions of everything.

Reply:
Yeah, there's a lot to fix with the default configuration before we turn this on. I'll go through the dependencies it wants to update and try to form an opinion about each. We might end up with a whitelist with just a few dependencies being updated automatically.